### PR TITLE
13.0 mig purchase discount

### PR DESCRIPTION
**Pull request by sa3m**
Standard migration

### CODE REVIEW

**Comment by kongrattapong** [comment]
Functional  review :+1: 

but please fix the travis, thank you

**Comment by sa3m** [comment]
It seems that the price_unit of move2 is wrong because the tax is included in the total (total_void returned by compute_all) that is compared with 161. @pedrobaeza I saw you worked on _get_stock_move_price_unit method that is related to this issue. Any clue on this ?

**Comment by pedrobaeza** [comment]
That should mean that methods have changed in v13 and you have to overwrite other things, but I can't say more without analyzing this in deep, which I'm not able to do it now.

**Comment by sa3m** [comment]
Thanks for the feedback.

**Comment by sa3m** [comment]
I figured it out. You have to set invoice_repartition_line_ids with account_id in tax and it seems that you cannot set cost_method on product now. But still, the checks failed for reasons out of my comprehension.

**Comment by sa3m** [comment]
This was an error from pip in travis. They patched pip so now it works. But still the ci/runbot is in error, how does this work ? I don't see any error. Thanks

**Comment by sergio-teruel** [comment]
Tested locally and Ok..

**Comment by younismahsud** [comment]
When this merge request is going to be merged with branch 13.0?

File: purchase_discount/report/purchase_report.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+from odoo import fields, models
+
+import odoo.addons.decimal_precision as dp
+
+
+class PurchaseReport(models.Model):
+    _inherit = "purchase.report"
+
+    discount = fields.Float(
+        string="Discount (%)", digits=dp.get_precision("Discount"), group_operator="avg"
```
Comment (juppe) [inline purchase_discount/report/purchase_report.py]: This gives a "Deprecated call to decimal_precision.get_precision" warning in Odoo
```suggestion
        string="Discount (%)", digits="Discount", group_operator="avg"
```

The `import odoo.addons.decimal_precision as dp` can also be removed when this is fixed.

Comment (sa3m) [reply purchase_discount/report/purchase_report.py]: Done ! Thanks for the feedback

**Comment by andres-pcg** [comment]
Hey, can we have this merged? @juppe

**Comment by juppe** [comment]
> Hey, can we have this merged? @juppe

I cannot comment on that, as I'm not a member of the OCA team. But hopefully soon. I'm also eagerly waiting for this to be merged.

**Comment by andres-pcg** [comment]
@pedrobaeza can you help?

**Comment by MarkingSC** [comment]
Is it ready? Hope this to be merged soon... @pedrobaeza can you do it?
@sa3m thanks for migrating

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**